keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -46,8 +46,11 @@ class SqlArtifactRepository(
       .set(DELIVERY_ARTIFACT.UID, randomUID().toString())
       .set(DELIVERY_ARTIFACT.NAME, artifact.name)
       .set(DELIVERY_ARTIFACT.TYPE, artifact.type.name)
```

Comment:
It's probably worth giving this file a close look because I'm not a sql expert. There's lots of changes in the sql artifact repository because we're now storing versions for artifacts separate from the "delivery artifact".